### ISSUE TITLE
[REQ] Adding Yandex and DeepL as translators

### ISSUE THREAD

**Issue by Farooq87** [Dadangdut33/Speech-Translate#74]
Thanks for your great work and this project. Nothing I'm frustrated with. Just ask you kindly to add as additional translators yandex and Deepl please.  Both have very good quality of AI too and free to use. Issue with them is they stop recording after 5 seconds of silence. And your great app doesn't stop

**Comment by FelicityRooting** [comment]
Yes, if the owner can add the DeepL translator, that will be great. DeepL is much more better than Google translator

**Comment by Farooq87** [comment]
> Yes, if the owner can add the DeepL translator, that will be great. DeepL is much more better than Google translator
However DeepL can transcribe only 4 languages from voice. Yandex can recognize many languages in voice to text
